aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -541,6 +541,39 @@ static void checkBasicAsStridedValidForSlice(
       "rewrite the `as_strided` call as a sequence of PyTorch view operations");
 }
 
+Tensor _new_zeros_with_same_meta_batching_rule(
```

Comment:
I think this function assumes that self is batched and other is not? We should check that at the beginning to be sure.